bullet-train.zsh-theme

```diff
@@ -13,6 +13,9 @@
 # The default configuration, that can be overwrite in your .zshrc file
 # ------------------------------------------------------------------------------
 
+# space variable to make spaces more visible and avoid bugs
+space=' '
```

Comment:
It should be considered a work in progress and very much would appreciate suggestions how to avoid variable space pollution. I am not very good at shell scripting :(